README.md

```diff
@@ -12,67 +16,100 @@ of your source files for more accurate results.
 
 ![](https://raw.githubusercontent.com/realm/SwiftLint/main/assets/screenshot.png)
 
-This project adheres to the [Contributor Covenant Code of Conduct](https://realm.io/conduct).
+This project adheres to the
+[Contributor Covenant Code of Conduct](https://realm.io/conduct).
 By participating, you are expected to uphold this code. Please report
 unacceptable behavior to [info@realm.io](mailto:info@realm.io).
 
-> Language Switch: [中文](https://github.com/realm/SwiftLint/blob/main/README_CN.md), [한국어](https://github.com/realm/SwiftLint/blob/main/README_KR.md).
+> Switch Language:
+> [中文](https://github.com/realm/SwiftLint/blob/main/README_CN.md)
+> [한국어](https://github.com/realm/SwiftLint/blob/main/README_KR.md)
+
+## Video Introduction
+
+To get a high-level overview of SwiftLint, we encourage you to watch this
+presentation recorded January 9th, 2017 by JP Simard (transcript provided):
+
+[![Presentation](https://raw.githubusercontent.com/realm/SwiftLint/main/assets/presentation.svg)](https://youtu.be/9Z1nTMTejqU)
 
 ## Installation
 
-### Using [Swift Package Manager](https://github.com/apple/swift-package-manager):
```

Comment:
I prefer the clickable name over the explicit URL.

Reply:
This was something that Garric and I discussed a bit before proposing this new style. And I have just gone back and looked at the new rendered markdown again for a fresh perspective. I do still believe this new style is preferred for the following reasons:

- The word "Using" does not fit with all of the headings, so there would be a lack of consistency when other words are used on some of them.
- Not all have URLs, so again from a consistency perspective, some of the headings would have different colors than others, due to being clickable.
- Seeing the URLs provides readable metadata that is helpful when looking at each of them.

Reply:
The thing is that bare URL are discouraged by style guides and (Markdown) linters and I also consider them as a distraction while reading. Differently colored headlines are okay I think. These links are anyway not that important but rather friendly pointers.

Reply:
Oh, thanks for that clarification 👍

I like the idea of removing "Using" so the entire headline can be clickable.